src/regressionmodel.jl

```diff
@@ -117,3 +117,27 @@ Return the offset used in the model, i.e. the term added to the linear predictor
 known coefficient 1.
 """
 function offset end
+
+"""
+    linearpredictor(model::RegressionModel)
+
+Return the model's linear predictor, `Xβ` where `X` is the model matrix and `β` is the
+vector of coefficients, or `Xβ + offset` if the model was fit with an offset.
+"""
+function linearpredictor(model::RegressionModel)
+    X = modelmatrix(model)
+    β = coef(model)
+    η = X * β
```

Comment:
Are we sure this will always be correct? That sounds reasonable but I wonder whether there may be weird model families where this wouldn't apply.

Reply:
That's possible, though I don't know of any uses of the term that don't refer to this formulation. People with weird models can always define a more specific method than this fallback.

Reply:
My worry was that if this isn't correct for all models, this fallback will return incorrect results, which is much worse than having no fallback.

Reply:
Fair. I've removed the definition and left it as a stub.